catroid/build.gradle

```diff
@@ -112,6 +114,7 @@ android {
             buildConfigField "String", "PROJECT_NAME", "\"No Standalone Project\""
             buildConfigField "boolean", "FEATURE_APK_GENERATOR_ENABLED", "false"
             buildConfigField "boolean", "CRASHLYTICS_CRASH_REPORT_ENABLED", "true"
+            buildConfigField "boolean", "FIREBASE_CLOUD_MESSAGING_ENABLED", "true"
```

Comment:
Please rename to `"FEATURE_CLOUD_MESSAGING_ENABLED"`
`FEATURE…` to be consistent with all other flags.
~`FIREBASE…`~ because it's not important what service we're using ^^. Previously FCM was called GCM and we would have need to update that everywhere if it would've been implemented already. It also feels more abstract :wink: